ext/reflection/php_reflection.stub.php

```diff
@@ -254,16 +254,16 @@ public function hasProperty(string $name) {}
     public function getProperty(string $name) {}
 
     /** @return ReflectionProperty[] */
-    public function getProperties(?int $filter = null) {}
+    public function getProperties(?int $modifiers = null) {}
```

Comment:
Other possibilities: $filtered_modifiers, $only_modifiers

Reply:
```
$rc->getProperties(filter: ReflectionProperty::IS_PUBLIC);
$rc->getProperties(only: ReflectionProperty::IS_PUBLIC);
$rc->getProperties(modifiers: ReflectionProperty::IS_PUBLIC);
$rc->getProperties(filter_modifiers: ReflectionProperty::IS_PUBLIC);
$rc->getProperties(only_modifiers: ReflectionProperty::IS_PUBLIC);
```
Hm ... my preference would go to the original `$filter` followed by `$filter_modifiers`.

Reply:
I have no hard feelings about any of the names :) So I'm changing this.